homeassistant/components/flume/sensor.py

```diff
@@ -27,68 +38,77 @@
         vol.Required(CONF_PASSWORD): cv.string,
         vol.Required(CONF_CLIENT_ID): cv.string,
         vol.Required(CONF_CLIENT_SECRET): cv.string,
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_NAME): cv.string,
     }
 )
 
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
We can't use the async api from sync context. Change this to `async def async_setup_platform`.

Reply:
And `add_entities` to `async_add_entities`.

Reply:
🤦‍♂ Thanks.

Reply:
https://github.com/home-assistant/core/pull/33855